jobs/kpi-forecasting/ci_job.yaml

```diff
@@ -11,5 +11,5 @@ build-job-kpi-forecasting:
         name: Build Docker image
         command: docker build -t app:build jobs/kpi-forecasting/
     - run:
-          name: Test Code
-          command: docker run app:build pytest --black
+        name: Test Code
+        command: docker run app:build pytest --black
```

Comment:
```suggestion
        command: docker run app:build black .
```

Reply:
I don't understand why, but the `build-docker-etl` check is still failing after updating from master and making this change.